src/main/java/frc/robot/Constants.java

```diff
@@ -77,6 +77,12 @@ public static class ControllerConstants{
     public static int firstAuto = 12;
     public static int secondAuto = 10;
     public static int thirdAuto = 8;
+
+    public static int OpenClaw = 2;
+    public static int CloseClaw = 3;
+    public static int RetractArms = 4;
+    public static int ExtendIntake = 5;
+    public static int RetractIntake = 6;
```

Comment:
When naming variables in java you typically use camelCase as opposed to PascalCase. You can read more about that here: https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/